build-tools/pom.xml

```diff
@@ -0,0 +1,12 @@
+<project>
+  <!--
+    build-tools contains build resources referenced by sub-projects. It follows the
+    "Multimodule Configuration" described here:
+      https://maven.apache.org/plugins/maven-checkstyle-plugin/examples/multi-module-config.html
+  -->
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.beam</groupId>
+  <artifactId>build-tools</artifactId>
+  <version>1.0</version>
```

Comment:
build-tools will be released, having a fixed version won't work in the long run, swap to use
${project.version}

We should use ${project.version} everywhere in our version declarations and intramodule deps instead of a fixed version. Only the parent pom should declare something else.